README.md

```diff
@@ -29,7 +29,7 @@ git clone https://github.com/your-username/baby-bliss-bot
 cd baby-bliss-bot
 ```
 
-### Create/Activitate Virtual Environment
+### Create/Activiate Virtual Environment
```

Comment:
The fix is wrong :-(.  There is an extra "i" before the final "a".
Activiate
Activate

Reply:
Good catch. Sorry I cannot spell. :) Now fixed.